testnets/holesky/README.md

```diff
@@ -0,0 +1,791 @@
+
+BOLT_SIDECAR_CONSTRAINTS_API = "http://cb_pbs:18550"     # The address of the PBS module (static)
+BOLT_SIDECAR_BEACON_API = ""
+BOLT_SIDECAR_EXECUTION_API = ""
+BOLT_SIDECAR_ENGINE_API = ""                             # The execution layer engine API endpoint
+BOLT_SIDECAR_JWT_HEX = ""                                # The engine JWT used to authenticate with the engine API
+BOLT_SIDECAR_BUILDER_PROXY_PORT = "18551"                # The port on which the sidecar builder-API will listen on. This is what your beacon node should connect to.
+BOLT_SIDECAR_FEE_RECIPIENT = ""                          # The fee recipient
+BOLT_SIDECAR_VALIDATOR_INDEXES = ""                      # The active validator indexes (can be defined as a comma-separated list, or a range)
+                                                         # e.g. "0,1,2,3,4" or "0..4", or a combination of both
+```
+
+To initialize commit-boost, run the following command:
+
+```bash
+commit-boost init --config cb-bolt-config.toml
+```
+
+This will create three files:
+
+- `cb.docker-compose.yml`: which contains the full setup of the Commit-Boost services
+- `.cb.env`: with local env variables, including JWTs for modules
+- `target.json`: which enables dynamic discovery of services for metrics scraping via Prometheus
+
+**Running**
+
+The final step is to run the Commit-Boost services. This can be done with the following command:
+
+```bash
+commit-boost start --docker cb.docker-compose.yml --env .cb.env
+```
+
+This will run all modules in Docker containers.
+
+> [!IMPORTANT] > `bolt-boost` will be exposed at `pbs.port` (18551 by default, set with `BOLT_SIDECAR_BUILDER_PROXY_PORT`), and your beacon node MUST be configured
+> to point the `builder-api` to this port for Bolt to work.
+
+### Observability
+
+Commit-Boost comes with various observability tools, such as Prometheus,
+cadvisor, and Grafana. It also comes with some pre-built dashboards, which can
+be found in the `grafana` directory.
+
+To update these dashboards, run the following command:
+
+`bash ./update-grafana.sh `
+
+In this directory, you can also find a Bolt dashboard, which will be launched
+alongside the other dashboards.
+
+# Register your validators on-chain on the Bolt Registry
+
+Once you are successfully running the Bolt sidecar you need to register on-chain
+on the Bolt Registry to successfully receive preconfirmation requests from users
+and RPCs. This step is needed to prove that your commitments are economically
+backed.
+
+In order to do that you need some collateral in the form of whitelisted Liquid
+Staking Token (LST) that needs to be restaked in either the Symbiotic or
+EigenLayer protocol.
+
+Then you need to interact with two contracts on Ethereum Mainnet:
+`BoltValidators` and `BoltManager`. The former is used to register your
+active validators into the protocol, while the latter is used to manage to
+register as an operator into the system and integrate with the restaking
+protocols.
+
```

Comment:
Let's add supported collaterals and their addresses per restaking protocol. We just support the eth derivatives for now.